pyproject.toml

```diff
@@ -47,6 +47,7 @@ dev = [
     "aviary.litqa",
     "fhaviary[image,llm,server,typing,xml]",
     "ipython>=8",  # Pin to keep recent
+    "jupyter>=1.0.0",  # For running notebooks
```

Comment:
I think this should be `ipykernel`, which is the more minimal set

Reply:
I'm planning to instruct users to run `jupyter notebook` to run the tutorials so I guess `ipykernel` wouldn't be sufficient for this?

Reply:
Okay gotchu. Yeah then feel free to keep it, resolving this one